messages.js

```diff
@@ -193,13 +193,13 @@ exports.init = function init(sbot) {
      */
     optsForSeed(mfKeys, mainfeedId, seed) {
       return {
-        recps: [mainfeedId],
-        encryptionFormat: 'box',
         content: {
           type: 'metafeed/seed',
           metafeed: mfKeys.id,
           seed: seed.toString('hex'),
         },
+        recps: [mainfeedId],
+        encryptionFormat: 'box2',
```

Comment:
:question:  do we want to lock ourselves to one encryptionFormat like this?

note that I don't want it to be `box` by default because I don't plan to use that in my apps.

Reply:
@arj03 Do you have opinions about encrypting the root mf seed as box1 or box2? I don't have an opinion, but I know you're the one who devised this solution.